### DISCUSSION TITLE
How do I reset the question asking if I want to connect to a workspace?

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#684] [🙋♀️TE3 Q&A] [answered]
I opened a BIM file locally and TE/3 asked me if I wanted to connect to a workspace. I accidentally clicked "Don't ask me again" and I want that question coming back. I cannot find in Preferences where to re-enable that.

**Comment by otykier** [comment, marked as answer]
Open the .tmuo file next to to the model in notepad, then remove the `"UseWorkspace": false,` line.

**Reply by edhans** [reply]
Ahh... I was thinking it was a "global" setting for future launches, not for that specific BIM. Thanks!

**Reply by JCAllegion** [reply]
I hope this is moved into TE3 and doesn't require going into a .tmuo file some time in the future.

**Reply by otykier** [reply]
> I hope this is moved into TE3 and doesn't require going into a .tmuo file some time in the future.

That’s the plan, although we don’t have a timeline for that yet.
